t/02-use.t

```diff
@@ -28,10 +28,12 @@ diag "no proj bin dir found via bin_dir method\n" if not @bin;
 my $info = eval {
     Alien::Proj4->load_projection_information
 };
-is $@, '', 'got projection information without error';
+my $e = $@;
+diag $e if $e;
+is $e, '', 'got projection information without error';
```

Comment:
This whole change doesn't seem to serve any purpose? The `is` will already show the message if it's not empty-string. That is why I put that there.

Reply:
One more thing I did not know/remember about is().   I'll remove it.